connect/runtime/src/main/java/org/apache/kafka/connect/tools/SchemaSourceTask.java

```diff
@@ -160,14 +161,22 @@ public void start(Map<String, String> props) {
             return result;
         } else {
             synchronized (this) {
-                this.wait();
```

Comment:
That's a good point.  Let's keep it simple here and just call `Throttler#throttle`.